plutus-benchmark/validation/BenchCek.hs

```diff
@@ -0,0 +1,7 @@
+module Main where
+
+import Common
+
+-- | benchmarks only for the CEK execution time of the data/*.flat validation scripts
+main :: IO ()
+main = validationBench CekOnly
```

Comment:
I was going to say that we could just have one program with a switch, but this is probably better because you don't have to remember how to deal with benchmarking options in cabal (which maybe doesn't work with the /benchmark thing anyway).